database/storeprocds/spGetAllTransactionType.sql

```diff
@@ -0,0 +1,26 @@
+-- =============================================
+-- Create basic stored procedure template
+-- =============================================
+
+-- Drop stored procedure if it already exists
+IF EXISTS (
+  SELECT * 
+    FROM INFORMATION_SCHEMA.ROUTINES 
+   WHERE SPECIFIC_SCHEMA = N'dbo'
+     AND SPECIFIC_NAME = N'spGetAllTransactionType' 
+)
+   DROP PROCEDURE dbo.spGetAllTransactionType
+GO
+
+CREATE PROCEDURE dbo.spGetAllTransactionType
+	
+AS
+	SELECT RecId, Name, Description FROM TRANSTYPE 
```

Comment:
if table field is same as sys reserved keyword so write like this [Name], [Description]

Reply:
fixed